### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sul-devops-team**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@
   # Print the 10 slowest examples and example groups at the
   # end of the spec run, to help surface which specs are running
   # particularly slow.
-  config.profile_examples = 10
+  # config.profile_examples = 10
```
Comment (aaron-collier) [inline spec/spec_helper.rb]: Is this change intentional and/or necessary?

Comment (ndushay) [reply spec/spec_helper.rb]: it's not necessary, it is intentional.  IMO, there is no need to profile stuff unless the tests run slow, and it clutters up the test results in my terminal.  Especially when fixing bugs.
